File: TransCoder_tokenized_test_set_functions/cpp/GIVEN_AN_ARRAY_OF_PAIRS_FIND_ALL_SYMMETRIC_PAIRS_IN_IT.cpp
void findSymPairs ( int arr [ ] [ 2 ], int row ) {
  unordered_map < int, int > hM;
  for ( int i = 0;
  i < row;
  i ++ ) {
    int first = arr [ i ] [ 0 ];
    int sec = arr [ i ] [ 1 ];
    if ( hM . find ( sec ) != hM . end ( ) && hM [ sec ] == first ) cout << "(" << sec << ", " << first << ")" << endl;
    else hM [ first ] = sec;
  }
}

File: TransCoder_tokenized_test_set_functions/python/GIVEN_AN_ARRAY_OF_PAIRS_FIND_ALL_SYMMETRIC_PAIRS_IN_IT.py
def findSymPairs ( arr, row ) :
    hM = dict ( )
    for i in range ( row ) :
        first = arr [ i ] [ 0 ]
        sec = arr [ i ] [ 1 ]
        if ( sec in hM . keys ( ) and hM [ sec ] == first ) :
            print ( "(", sec, ",", first, ")" )
        else :
            hM [ first ] = sec
    
